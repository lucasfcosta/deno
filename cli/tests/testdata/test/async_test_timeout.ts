Deno.test("test 1", () => {});

Deno.test("test 2", async () => {
  await new Promise(() => {});
});

Deno.test("test 3", () => {});
