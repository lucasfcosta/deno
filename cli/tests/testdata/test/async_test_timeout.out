Check [WILDCARD]/testdata/test/async_test_timeout.ts
running 3 tests from [WILDCARD]/testdata/test/async_test_timeout.ts
test test 1 ... ok ([WILDCARD])
test test 2 ... FAILED ([WILDCARD])
test test 3 ... ok ([WILDCARD])

failures:

test 2
Error: Test took longer than the 5000ms timeout.
    at deno:runtime/js/40_testing.js:421:18
    at Object.action (deno:ext/timers/01_timers.js:161:11)
    at handleTimerMacrotask (deno:ext/timers/01_timers.js:78:12)

failures:

	test 2

test result: FAILED. 2 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])

error: Test failed
